examples/plugin/src/main.rs

```diff
@@ -0,0 +1,38 @@
+use plugin_api::plugin_api as pylib_module;
+use pyo3::prelude::*;
+use pyo3::types::PyList;
+use std::path::Path;
+
+fn main() -> Result<(), Box<dyn std::error::Error + Send + Sync>> {
+    //"export" our API module to the python runtime
+    pyo3::append_to_inittab!(pylib_module);
+    //spawn runtime
+    pyo3::prepare_freethreaded_python();
+    //import path for python
+    let path = Path::new("./");
+    //do useful work
+    Python::with_gil(|py| {
+        //add the current directory to import path of Python (do not use this in production!)
+        let syspath: &PyList = py.import("sys")?.getattr("path")?.extract()?;
+        syspath.insert(0, &path)?;
+        println!("Import path is: {:?}", syspath);
+
+        // Now we can load our plugin.py file
+        let plugin = PyModule::import(py, "plugin")?;
```

Comment:
You can just call the initialization function, and avoid messing with sys.path entirely:
```rust
let module = PyModule::new(py, "plugin")?;
plugin_api::plugin_api(py, &module)?;
```

Reply:
No, this will not work as expected, since plugin.py will not land on the import path. And then, if plugin.py has any dependencies of its own, they will fail to import which will confuse the user until they figure out they need to mess with import path for it to work as you'd expect Python to work. I will actually add some deps to plugin.py to illustrate this.